rxandroidble/src/main/java/com/polidea/rxandroidble2/internal/util/RxBluetoothManagerWrapper.java

```diff
@@ -0,0 +1,22 @@
+package com.polidea.rxandroidble2.internal.util;
+
+import android.bluetooth.BluetoothDevice;
+import android.bluetooth.BluetoothManager;
+
+import java.util.List;
+
+import bleshadow.javax.inject.Inject;
+
+public class RxBluetoothManagerWrapper {
+
+    private final BluetoothManager bluetoothManager;
+
+    @Inject
+    public RxBluetoothManagerWrapper(BluetoothManager bluetoothManager) {
+        this.bluetoothManager = bluetoothManager;
+    }
+
+    public List<BluetoothDevice> getConnectedDevices(int profile) {
+        return bluetoothManager.getConnectedDevices(profile);
+    }
```

Comment:
Could we make this `getConnectedPeripherals()`?
This would make it easier to potentially mock this class since we would not need to have `BluetoothProfile` class outside of BluetoothManagerWrapper.
```suggestion
    public List<BluetoothDevice> getConnectedPeripherals() {
        return bluetoothManager.getConnectedDevices(BluetoothProfile.GATT_SERVER);
    }
```

Reply:
Yup, good idea - I was conflicted when making the change. Peripherals make more sense. Naming changed to properly reflect this